src/bikeracks/templates/scripts.js

```diff
@@ -48,6 +49,13 @@ class BikeMap {
             this.tempMarker.remove();
         }.bind(this));
         
+        // suggest removal click handler
+        this.$suggestRemoval.on('click', function(e) {
+            const button = $(event.target),
```

Comment:
Where is `event` defined? Did you mean `e` ?

Reply:
Yes.